### ISSUE TITLE
Cannot input  modification characters correctly

### ISSUE THREAD

**Issue by rugk** [mike-fabian/ibus-typing-booster#36]
The input in gedit or so works, but in Firefox ' cannot be added in the text. Instead of that char, it just adds a space.

Same for ^, \` (that's the char you use to start/end an inline code in GitHub's markdown – `example`), BTW.

I don't know how it works in other keyboard layouts, but here is how I have to write them in the German (Qwertz) layout:
1. Press that key, i.e. '. Nothing appears.
2. Prewss space afterwards

The reason these accents/cirumflex, … chars need a space for confirmation is that you can also write á or so with them. (If you do so, you just press a after you press \` or so, instead of a space)

E.g. even gedit suffers from some things. Here is what I can(not) write:

char | Firefox | gedit
-- | ----- | ----
` | fails | works  
' | fails | works
^ | fails | works
ê | fails | fails
é | fails | fails
è | fails | fails

So basically I cannot…
* write the ones, where I need to press space afterwards, end up as a space only and only affect Firefox
*  the ones where I press any other character afterwards to modify that character, end up as the original character. This affects gedit and Firefox.

**Comment by mike-fabian** [comment]
Are you using t-latn-post, t-latn-pre, or native keyboard input?

**Comment by rugk** [comment]
native

**Comment by mike-fabian** [comment]
Apparently you are  using a German keyboard layouts where ', ^, ... are dead keys.

It seems like ibus-typing-booster cannot handle these  dead keys well, I need  to investigate this.

In the mean time, I recommend to use the German keyboard without dead keys
(This is called “Deutsch (ohne Akzenttasten)” in the gnome-control-center) and use
Latin-Post or Latin-Pre in ibus-typing-booster to be able to type the accents.

**Comment by mike-fabian** [comment]
Latn-Pre behaves  quite similar to the dead keys.

**Comment by mike-fabian** [comment]
By the way, I usually use both Latin-Post *and* direct keyboard input at the same time with ibus-typing-booster (and a keyboard layout without dead keys).  You can use up to 10 input methods at the same time with ibus-typing-booster, see:

http://mike-fabian.github.io/ibus-typing-booster/documentation.html#multilingual-input

Currently setting up an input method like  Latin-Post plus direkt keyboard input is easy by
using the option “add direct keyboard input”. Configuring other combinations of input methods
is currently a bit more difficult because it can only be done using the command line, see:

http://mike-fabian.github.io/ibus-typing-booster/documentation.html#multilingual-setup-dconf

But I am almost  finished rewriting the setup UI to make setup of multiple input methods
and multiple dictionaries easy using the graphical setup tool.

**Comment by mike-fabian** [comment]
The [2.7.0 release](https://github.com/mike-fabian/ibus-typing-booster/releases/tag/2.7.0)
has support for  Compose  now. Please try again, your German keyboard layout with dead keys
should work now with ibus-typing-booster.
